### ISSUE TITLE
cnetpush - which Isis3 'nightly' version do we test on??

### ISSUE THREAD

**Issue by ascbot** [DOI-USGS/ISIS3#751]
---

Author Name: **Tammy Becker** (Tammy Becker)

Original Assignee: Steven Lambright

---

I am hearing in the "hallways" that we are supposed to be testing a version of Isis3 called isis3cnetpush.....this has confused me (which isn't hard to do), it's definitely 'my bad' if I missed an email!?  I have been testing under isis3nightly, but now have heard that fixes and testing should be performed on isis3cnetpush??  (I have seen some 'qnet' emails from Tracie about isis3cnetpush, but I am not clear if this is just for qnet or all binary control apps?).

I hoped that this would be a good place to post so that someone please clarify what the 'formal' plan is?   

Thank you much!

**Comment by ascbot** [comment]
---

Original Redmine Comment
Author Name: **Stuart Sides** (@scsides)
Original Date: 2011-07-12T15:18:10Z

---

There has been some understandable confusion on what version of ISIS
needs to be used for testing during the cnet push. To help clear this
up, we are removing the "cnetpush" version of ISIS and merging all
cnetpush changes into the current nightly. These changes will be
available in isis3nightly2011-07-12.

To provide a fall back in case the nightly build fails we have frozen
the current nightly (isis3nightly2011-07-09) and provided a link to it
called isis3frozen. This version will remain available until we get the
go ahead from you to remove it. This should happen when we create the
next stable production version be that "Astro", "LMMP" or something else.

cnetpush testing of applications should be done using the nightly build.
If you need to test the way it was before we started the push use
isis3frozen. All ISIS 3 software changes from here on will be available
in the next day's nightly build. No changes will be made to "isis3frozen"

Disposition of current versions of ISIS3:

isis3frozen (isis3nightly2001-07-09) - will remain as is until no longer needed
isis3nightly2011-##-## - will continue to keep the last 21 versions
isis3nightly2001-06-08 - will remain as is until no longer needed 
isis3cnetpush - will be deleted in the morning (7/12/11)
isis3astro - will remain as is with required patches if required
isis3.2.1 - will remain as is with required patches if required


If anything here needs more explanation please let me know.

**Comment by ascbot** [comment]
---

Original Redmine Comment
Author Name: **Tammy Becker** (Tammy Becker)
Original Date: 2011-07-14T03:59:36Z

---

Can someone from the development group add a brief description of the versions, whatever is significant that makes them different from one another?  For instance, I know that isis3astro = "non-binary" or "pre-binary"  or isis3.2.1 = "current public release version"....the isis3nightly's are??

isis3frozen (isis3nightly2001-07-09) - will remain as is until no longer needed
isis3nightly2011-##-## - will continue to keep the last 21 versions
isis3nightly2001-06-08 - will remain as is until no longer needed 
isis3cnetpush - will be deleted in the morning (7/12/11)
isis3astro - will remain as is with required patches if required
isis3.2.1 - will remain as is with required patches if required

**Comment by ascbot** [comment]
---

Original Redmine Comment
Author Name: **Tammy Becker** (Tammy Becker)
Original Date: 2011-07-26T19:56:41Z

---

Stuart just explained the following:

isis3astro2011-06-09 =  The last isis3astro version without any control network binary and keyword changes.

This version is similar to the public 'isis3.2.1', except that the public version probably has more recent patches.

**Comment by ascbot** [comment]
---

Original Redmine Comment
Author Name: **Steven Lambright** (Steven Lambright)
Original Date: 2012-01-26T19:51:05Z

---

I don't believe this issue pertains to what we're doing for Isis builds any more.
